### PR TITLE
First pass at Adding the timeseries API

### PR DESCRIPTION
**Pull request by goldblatt**
This PR addresses issue #210 

### Please Check
- [] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [?] Are tests passing?

### CODE REVIEW

File: libs/datasets/can_model_output_schema.py [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
+    DEAD,
+    BEDS,
     "i",
     "j",
     "k",
     "l",
-    "population",
+    POPULATION,
     "m",
     "n",
```
Comment (goldblatt) [inline libs/datasets/can_model_output_schema.py:55]: Generally confused about what these are? someone pointed me to a few places and i looked at the original file, but where lines 18-32 came from

File: libs/functions/generate_api.py [hunk 2]
```diff
@@ -4,11 +4,18 @@
...
+    CANPredictionTimeseriesRow,
     _Projections,
     _ResourceUsageProjection,
 )
 from libs.datasets import results_schema as rc
 from libs.constants import NULL_VALUE
+from libs.functions import get_can_projection
+from libs.datasets.dataset_utils import AggregationLevel
+from libs.us_state_abbrev import US_STATE_ABBREV
+from libs.datasets import can_model_output_schema as can_scema
```
Comment (ghop02) [inline libs/functions/generate_api.py:18]: can_scema -> can_schema

File: libs/datasets/can_model_output_schema.py [hunk 3]
```diff
@@ -1,29 +1,59 @@
 """
 Ouput of the 
 """
+DATE = "date"
+TOTAL = "total"
+SUSCEPTIBLE = "susceptible"
+EXPOSED = "exposed"
+INFECTED = "infected"
+INFECTED_A = "infected_a"
```
Comment (ghop02) [inline libs/datasets/can_model_output_schema.py:9]: would be cool to name this closer to the comments below

File: libs/datasets/can_model_output_schema.py (hunk 1 above)
Comment (igorkofman) [reply libs/datasets/can_model_output_schema.py:55]: they are just unused values from old spreadsheet

**Pull request merged**